### ISSUE TITLE
Angular Material: Tooltip not properly positioned if gantt is shown in a mdBottomSheet

### ISSUE THREAD

**Issue by kschwidder** [angular-gantt/angular-gantt#374]
Context:
the gantt is included in a mdButtonSheet and tooltips are active.

Behaviour:
When the tooltip is activated it is not shown at the corresponding task but approx. 100 px below.

When the tooltip is activated on Gigi Schwidder than it is placed as follows.
This behavior does not occur if the gantt is embedded in the main content.

![gantttooltip](https://cloud.githubusercontent.com/assets/3265624/6426943/a1ac6d14-bf71-11e4-9a44-2c05aa56c8d3.png)

**Comment by Toilal** [comment]
Tooltip plugin is really weird ... I don't know why this occurs.

@abachborius2 wrote an internal tooltip plugin based on [JQuery Tooltipster](http://iamceege.github.io/tooltipster/) that fix many issues related to tooltips, maybe he can help for this issue.

**Comment by Gompje** [comment]
@kschwidder did you find a solution for this? I have the same issue; don't use mdButtonSheet though. 

When I inspect the tooltips it's the fixed positioning on the element itself that causes the culprit (eg: left and top css setting). When I remove those in the inspector the tooltip is positioned correctly.

**Comment by kschwidder** [comment]
@gompje: I have disabled tooltips for my gantt in the mdBottomSheet as it does not make sense to show them in my specific case. When I use mdDialog instead they are working right. 
Thanks for your investigation with the css settings. Will have a look at it.

**Comment by Toilal** [comment]
@gompje @kschwidder: Can we make something (CSS?) in mainstream that would solve this issue for material users ? Doesn't this issue still exists in latest angular-material version ?

**Comment by kschwidder** [comment]
@Toilal: The issue still exists if dialogs are used. It seems that the coordinates are differently managed.
Would it be suitable to change the tooltip such that it is displayed according to absolute coordinates where the mouse is ? 
The detection of the task is already correct and using the current physical mouse location when drawing the tooltip would simplify the plugin too.

**Comment by AlexFreem** [comment]
Hello. I have found same problem but in my case it works not only with tooltips or some other plugins. If i include [ngMaterial](https://material.angularjs.org/latest/#/) module to my app - all columns and task collapses.

Without ngMaterial, my app look like:
![Correct table](http://i.imgur.com/36V0tiM.png)

with included ngMaterial : 
![Incorrect table](http://i.imgur.com/gchcDwh.png)

**Comment by Toilal** [comment]
What do you mean by included ? You only load the ngMaterial module in the webapp, and it fails to display all the tasks ?

**Comment by AlexFreem** [comment]
@Toilal sorry for my bad English :)
Yes, it works as you commented.

**Comment by kschwidder** [comment]
Have not seen this in our ngMaterial app. Everything is working well upto master :-)

**Comment by Toilal** [comment]
Yes, i've created a ngMaterial app some months ago, and it worked well too.

**Comment by Toilal** [comment]
@kschwidder: I don't wan't to display the tooltip at mouse position, because it could hide the task title while hovering ...

**Comment by kschwidder** [comment]
Might be but when I add a gantt within md-dialog the tooltip is misplaced totally. In a standard html page it works properly. I seems that the coordinates of md-dialog are different. I hiding the task title so important as an issue ? When I hover over a task I would like to see the tooltip. Although I can show the task title also in the tooltip :-)

**Comment by Toilal** [comment]
I know, but currently this issue only occurs in ng-material, so it's not really blocking for most users. And it's still possible to copy/paste sources of tooltip plugin to write it's own.

If you have some time, it would be great to fork the master plunkr (link in README) and try to reproduce it. But I mark this one as ready on [HuBoard](https://huboard.com/angular-gantt/angular-gantt), so that means it's top priority :)

**Comment by icarballo** [comment]
I have the same problem with angular gantt and material tabs :|

**Comment by bentrm** [comment]
I had a similar problem, though it might be unrelated to the case described above:

Tooltips would be misplace if a custom component is loaded via the plugins `content` attribute. I copied the tooltip template and added the content directly using the same component as before. By loading this customized template via the plugins `template-url` attribute the tooltips are place reliably.

**Comment by medou-boushab** [comment]
Hello, I have the same issue. I'm displaying the angular gantt on a modal.
Does anyone have a solution for this?
